common/src/main/java/com/futurewei/common/model/SecurityGroupRule.java

```diff
@@ -0,0 +1,121 @@
+/*
+MIT License
+Copyright(c) 2022 Futurewei Cloud
+
+    Permission is hereby granted,
+    free of charge, to any person obtaining a copy of this software and associated documentation files(the "Software"), to deal in the Software without restriction,
+    including without limitation the rights to use, copy, modify, merge, publish, distribute, sublicense, and / or sell copies of the Software, and to permit persons
+    to whom the Software is furnished to do so, subject to the following conditions:
+
+    The above copyright notice and this permission notice shall be included in all copies or substantial portions of the Software.
+
+    THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+    FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY,
+    WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+*/
+package com.futurewei.common.model;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.hazelcast.nio.ObjectDataInput;
+import com.hazelcast.nio.ObjectDataOutput;
+import com.hazelcast.nio.serialization.IdentifiedDataSerializable;
+import lombok.EqualsAndHashCode;
+import lombok.Getter;
+import lombok.Setter;
+import lombok.ToString;
+import org.springframework.data.annotation.Id;
+
+import java.io.IOException;
+import java.io.Serializable;
+
+@Getter
+@Setter
+@EqualsAndHashCode
+@ToString
+public class SecurityGroupRule implements IdentifiedDataSerializable, Serializable {
+
+    private static final long serialVersionUID = 6529685098267757690L;
+
+    @Id
+    private String id;
+
+    @EqualsAndHashCode.Exclude
+    @JsonProperty("securitygroupid")
+    private String securityGroupId;
+
+    @EqualsAndHashCode.Exclude
+    @JsonProperty("remotegroupid")
+    private String remoteGroupId;
+
+    @EqualsAndHashCode.Exclude
+    @JsonProperty("direction")
+    private String direction;
+
+    @EqualsAndHashCode.Exclude
+    @JsonProperty("remoteipprefix")
+    private String remoteIpPrefix;
```

Comment:
is the remoteIpPrefix like '10.0.0.0/16' ? Ana ArionAgent split it by a backsplash